opennms-web-api/src/main/java/org/opennms/web/utils/QueryParameters.java

```diff
@@ -30,7 +30,7 @@
 
 public class QueryParameters {
 
-    private static final Integer DEFAULT_LIMIT = 10;
+    private static final Integer DEFAULT_LIMIT = 100;
```

Comment:
Does this have other implications?

Reply:
It affects other drop downs which have the incorrect limit. which aren't mentioned in the original "quick win" ticket, eg "Home / Admin / Configure SNMP by IP". I will do an investigation into other implications.